spec/extensions/index.rst

```diff
@@ -7,4 +7,5 @@ Extensions
    :caption: Extensions
    :maxdepth: 3
 
+   fast_fourier_transform_functions
```

Comment:
Minor: can we drop the `fast_` here? That's really an implementation detail (we don't care how fast it is:)).